oeps/oep-0019-bp-developer-documentation.rst

```diff
@@ -52,6 +52,21 @@ This proposal focuses on long-term developer *Documentation* needs. It does not
 Decisions
 *********
 
+* We will use `reStructuredText (rST)`_ for all developer documentation (including in docstrings).  See the :ref:`Format Rationale` section below for more info.
+
+* The `Developer Docs`_ section of docs.edx.org will serve as the index for all repo specific and system wide docs.
```

Comment:
1. Minor suggestion. I like the repetition here since `docs.edx.org` deals with multiple types of docs, but you may not agree.
```suggestion
* The `Developer Docs`_ section of docs.edx.org will serve as the index for all repo specific and system wide developer docs.
```
2. What is meant by "index"? Does this mean top-most landing page? Other? Also, is this for "public" documentation only? Is there another top-most landing page for private docs?

Reply:
I think I pulled those words from elsewhere, but it's meant to link back to any other repo or system specific docs.

Reply:
Precisely what that page provides is a little blurb and link to https://edx.readthedocs.io/projects/edx-developer-docs/en/latest/. I think a separate bullet about `edx-developer-docs`, which is really meant to be the index for developer docs, is probably more important. I think the docs page serves to get different personas to the various relevant home pages for their docs.